test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/DefaultSkipTokenHandlerTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if NETCORE
+using System;
+using Microsoft.AspNet.OData.Formatter.Serialization;
+using Microsoft.AspNet.OData.Test.Abstraction;
+using Microsoft.AspNet.OData.Test.Common;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+using Xunit;
+using ODataPath = Microsoft.AspNet.OData.Routing.ODataPath;
+#else
+using System;
+using System.Linq;
+using System.Web.Http.Routing;
+using Microsoft.AspNet.OData.Formatter;
+using Microsoft.AspNet.OData.Formatter.Serialization;
+using Microsoft.AspNet.OData.Test.Abstraction;
+using Microsoft.AspNet.OData.Test.Common;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+using Xunit;
+using ODataPath = Microsoft.AspNet.OData.Routing.ODataPath;
+#endif
+
+namespace Microsoft.AspNet.OData.Test.Query
+{
+    public class DefaultSkipTokenTests
+    {
+        [Theory]
+        [InlineData("http://localhost/Customers(1)/Orders", 10, null, false, "http://localhost/Customers(1)/Orders?$skip=10")]
+        [InlineData("http://localhost/Customers?$expand=Orders", 10, null, false, "http://localhost/Customers?$expand=Orders&$skip=10")]
+        public void GetNextPageLink_ReturnsCorrectNextLink(string baseUri, int pageSize, Object instance, bool enableSkipToken, string expectedUri)
```

Comment:
If the pageSize, instance, enableSkipToken are not changeable, we can move them directly into the test

Reply:
#Resolved